src/runtime_src/tools/xclbin/xclbindata.cxx

```diff
@@ -373,8 +389,13 @@ XclBinData::extractSectionData( int sectionNum, const char* name )
   else if ( header.m_sectionKind == BMC ) {
     extractAndWriteBMCImages((char*) data.get(), sectionSize);
     return true;
+  } else {
+    static unsigned int uniqueCount = 1;
+    type = "unknown(" + std::to_string(uniqueCount) + ")";
```

Comment:
Should this get incremented?

Reply:
Great catch.  Changes made to increment this value.